SonarQube.Client/Models/ServerExclusions.cs

```diff
@@ -57,5 +57,23 @@ public override string ToString()
                    "\n    Exclusions: " + string.Join(",", Exclusions) +
                    "\n    Global Exclusions: " + string.Join(",", GlobalExclusions);
         }
+
+        public override bool Equals(object obj)
+        {
+            return Equals((ServerExclusions)obj);
+        }
+
+        public bool Equals(ServerExclusions other)
+        {
+            if (ReferenceEquals(null, other)) return false;
+            if (ReferenceEquals(this, other)) return true;
+
+            return ToString().Equals(other.ToString());
+        }
+
+        public override int GetHashCode()
+        {
+            return base.GetHashCode();
```

Comment:
This is doing nothing in a confusing way.
We should provide a correct implementation or nothing (and we should provide a correct implementation rather than nothing) e.g. just return `this.ToString().GetHashCode()`;

Reply:
Sorry, missed that this was still a draft.

Reply:
Yep I'm adding it now 👍